cypress/e2e/basic-starter.cy.ts

```diff
@@ -130,7 +130,7 @@ describe('Taxonomy term page', () => {
   });
 
   it('should render places for Office (place type)', () => {
-    cy.visit('/taxonomy/term/13');
+    cy.visit('/place_type/office');
```

Comment:
These more reliable URLs are *not* part of Acquia CMS's starter kit. They're generated as part of the container build process.

Reply:
In general, it's a smart idea for the tests not to rely on serial IDs or UUIDs, since those can change each time our image is rebuilt.